lib/index.js

```diff
@@ -211,20 +212,37 @@ Segment.prototype.ampId = function(ctx) {
 
 Segment.prototype.send = function(path, msg, fn) {
   var url = 'https://' + this.options.apiHost + path;
-  var headers = { 'Content-Type': 'text/plain' };
   fn = fn || noop;
   var self = this;
 
   // msg
   msg = this.normalize(msg);
 
   // send
-  send(url, msg, headers, function(err, res) {
-    self.debug('sent %O, received %O', msg, arguments);
-    if (err) return fn(err);
-    res.url = url;
-    fn(null, res);
-  });
+  if (this.options.beacon && navigator.sendBeacon) {
+    // Beacon returns false if the browser couldn't queue the data for transfer
+    // (e.g: the data was too big)
```

Comment:
Gotcha - that's definitely more than enough for us (our messages are capped at 15kb). Just wanted to make sure we had an example relevant to our uses in the comment.